issue-9-body.md

```
@@ -0,0 +1,95 @@
+- [ ] フレームワーク検出システムの実装
+- [ ] テンプレートエンジンの構築
+- [ ] 各フレームワーク用テンプレート作成
+- [ ] コード生成ロジックの実装
+- [ ] 依存関係管理システム
+- [ ] フレームワーク固有の最適化ルール実装
+- [ ] 統合テストの作成
+
+## テンプレート設計
+
+### 構造例（Next.js）
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)